.golangci.yml

```diff
@@ -10,3 +10,9 @@ issues:
 linters:
   enable:
     - wrapcheck
```

Comment:
you need to add stylecheck here to enable that.

Reply:
Thanks!

I was looking at the options, I think these are actually enabled by default: https://golangci-lint.run/usage/linters/#enabled-by-default-linters

We just didn't notice them because some of these "problems" were introduced before the linter.